src/presenters/breadcrumbs-presenter.php

```diff
@@ -137,8 +137,17 @@ protected function crumb_to_link( $breadcrumb, $index, $total ) {
 			// If it's not the last element and we have a url.
 			$link      .= '<' . $this->get_element() . '>';
 			$title_attr = isset( $breadcrumb['title'] ) ? ' title="' . \esc_attr( $breadcrumb['title'] ) . '"' : '';
-			$link      .= '<a href="' . \esc_url( $breadcrumb['url'] ) . '"' . $title_attr . '>' . $text . '</a>';
-			$link      .= '</' . $this->get_element() . '>';
+			$link      .= '<a';
+
+			/**
+			 * This is needed because when the editor is loaded in an Iframe the link needs to open in a different browser window.
+			 * We don't want this behaviour in the front-end and the way to check this is to check if the block is rendered in a REST request. Thus being in the editor.
+			 */
+			if ( \wp_is_serving_rest_request() ) {
```

Comment:
This would mean that the attributes gets injected in the post content even when queried via REST API, which is something we don't want.